book/src/chap02.asciidoc

```diff
@@ -179,234 +178,232 @@ x = 5
+semântica::
+O significado de um programa.
 (((semantics)))
 
-semantic error::
-An error in a program that makes it do something other than what the programmer intended.
+erro de semântica::
+Um erro em um programa que faz com que ele faça algo diferente do que o programador pretendia.
 (((semantic error)))
 
 
-=== Exercises
+=== Exercícios
 
 [[ex02-1]]
-===== Exercise 2-2
+===== Exercício 2-2
 
-Repeating my advice from the previous chapter, whenever you learn a new feature, you should try it out in interactive mode and make errors on purpose to see what goes wrong.
+Repetindo o meu conselho do capítulo anterior, sempre que você aprende novas funcionalidades, você deve experimentá-las no modo interativo e cometer erros de propósito para ver o que acontece de errado.
 
-. We’ve seen that +n = 42+ is legal. What about +42 = n+?
+. Nos vimos que +n = 42+ é válido. E +42 = n+ ?
 
-. How about +x = y = 1+?
+. E que tal +x = y = 1+?
 
-. In some languages every statement ends with a semi-colon, +;+. What happens if you put a semi-colon at the end of a Julia statement?
+. Em algumas linguagens toda declaração acaba com um ponto e vírgula, +;+. O que acontece se você colocar um ponto e vírgula no final de uma declaração no Julia?
 (((pass:[;])))
 
-. What if you put a period at the end of a statement?
+. E se você quiser colocar um ponto no final de uma declaração ?
 
-. In math notation you can multiply +x+ and +y+ like this: +x y+. What happens if you try that in Julia? What about 5x?
+. Em notação matemática, você pode multiplicar +x+ e +y+ assim: +x y+. O que acontece se você tentar isso em Julia? E 5x?
 
 [[ex02-2]]
-===== Exercise 2-3
+===== Exercício 2-3
 
-Practice using the Julia REPL as a calculator:
+Pratique usando o REPL do Julia como uma calculadora:
 (((calculator)))
+. O volume de uma esfera com raio latexmath:[\(r\)] é latexmath:[\(\frac{4}{3} \pi r^3\)]. Qual é o volume de uma esfera de raio 5?
 
-. The volume of a sphere with radius latexmath:[\(r\)] is latexmath:[\(\frac{4}{3} \pi r^3\)]. What is the volume of a sphere with radius 5?
-
-. Suppose the cover price of a book is $ 24.95, but bookstores get a 40 % discount. Shipping costs $ 3 for the first copy and 75 cents for each additional copy. What is the total wholesale cost for 60 copies?
-
-. If I leave my house at 6:52 am and run 1 mile at an easy pace (8:15 per mile), then 3 miles at tempo (7:12 per mile) and 1 mile at easy pace again, what time do I get home for breakfast?
+. Suponha que o preço de cobertura de um livro é 24,95 R$, mas as livrarias possuem desconto de 40%. A entrega custa 3,00 R$ para a primeira cópia e 0,75 R$ para cada cópia adicional. Qual é o preço total do atacado para 60 cópias?
 
+. Se eu saio de casa às 6:52 da manhã e corro uma milha em um ritmo tranquilo (8:15 minutos por milha), em seguida 3 milhas em ritmo (7:12 minutos por milha) e 1 milha em um ritmo tranquilo novamente, a que horas eu chego em casa para o café da manhã?
```

Comment:
mude milha para quilômetros.

Reply:
8min15s e 7min12s

Reply:
@christianpeel if we localize miles to kilometers and change the times to make sense, we need to change something somewhere else? I don't see a chapter 2 on src/solutions.

Reply:
If you localize to kilometers in the .asciidoc files, I don't know that you need to change anything elsewhere.  I do not think there is a complete set of solutions in src/solutions, and I don't see a big need for to create a solutions file for chapter 2